Makefile

```diff
@@ -0,0 +1,3 @@
+#this will execute the program 
+all : 
+	gcc Hello\ World.c 
```

Comment:
what will be the name of the final executable?

Reply:
in this case, a.out

Reply:
can we create the executable as `obj/hello-world.c`?

Reply:
also we want to be able to do `make clean`
which will do rm -rf `obj/*`

Reply:
we also want `make clobber`
which will do `rm -rf obj`

Reply:
yes